src/Client/src/Asp.Versioning.Http.Client/System.Net.Http/HttpResponseMessageExtensions.cs

```diff
@@ -69,6 +70,67 @@ public static SunsetPolicy ReadSunsetPolicy( this HttpResponseMessage response )
         return policy;
     }
 
+    /// <summary>
+    /// Gets an API deprecation policy from the HTTP response.
+    /// </summary>
+    /// <param name="response">The <see cref="HttpResponseMessage">HTTP response</see> to read from.</param>
+    /// <returns>A new <see cref="DeprecationPolicy">deprecation policy</see>.</returns>
+    public static DeprecationPolicy ReadDeprecationPolicy( this HttpResponseMessage response )
+    {
+        ArgumentNullException.ThrowIfNull( response );
+
+        var headers = response.Headers;
+        var date = default( DateTimeOffset );
+        DeprecationPolicy policy;
+
+        if ( headers.TryGetValues( Deprecation, out var values ) )
+        {
+            var culture = CultureInfo.CurrentCulture;
+
+            foreach ( var value in values )
+            {
+                var split = value.Trim( '@' );
+                if ( long.TryParse( split, out var unixTimestamp ) )
+                {
+                    DateTimeOffset parsed;
+#if NETSTANDARD
+                    parsed = new DateTime(1970, 1, 1) + TimeSpan.FromSeconds(unixTimestamp);
```

Comment:
`DateTimeOffset.FromUnixTimeSeconds` is available in .NET Standard 1.3+ so this condition can be revised to `#if NETSTANDARD1_1` so that .NET Standard 2.0 takes the better path.